<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersPage</class>
 <widget class="QWizardPage" name="ParametersPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <widget class="SearchBar" name="searchBar" native="true"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="formContainer"/>
   </item>
   <item>
    <widget class="KMessageWidget" name="invalidParameterWarning">
     <property name="text">
      <string notr="true">Invalid parameter message</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="closeButtonVisible">
      <bool>false</bool>
     </property>
     <property name="messageType">
      <enum>KMessageWidget::MessageType::Error</enum>
     </property>
     <property name="icon">
      <iconset theme="QIcon::ThemeIcon::DialogError"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
  <customwidget>
   <class>SearchBar</class>
   <extends>QWidget</extends>
   <header>ui/misc/SearchBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
